### ISSUE TITLE
Adding a language to the Debian repository

### ISSUE THREAD

**Issue by patranger** [janet-lang/janet#970]
It would be very convenient if you would add packages to the Debian repository. Then programming in Janet would be more convenient because all updates could be done in one command.

**Comment by bakpakin** [comment]
Agreed. We need a Debian maintainer to do that, though. I don't think it would be all that hard to package since we can build and install with meson out of the box.

**Comment by bakpakin** [comment]
One of many tutorials online for doing this: https://blog.devgenius.io/how-to-build-debian-packages-from-meson-ninja-d1c28b60e709

I don't really use any debian based distros on the regular so I'm probably not the best person to do this, though

**Comment by patranger** [comment]
@bakpakin 
The link includes packaging instructions to deb format. I am happy to help with this, but I don't have time for this yet. If I find a moment, I'll take care of it.

https://www.intracto.com/en-be/blog/creating-debianubuntu-deb-packages

**Comment by bakpakin** [comment]
Thanks, @patranger. As well as packaging up Janet into a .deb, it might be nice to get it added to debian repository somewhere - I think we are at a level of stability appropriate for that.

**Comment by Matthewhughes3** [comment]
@patranger
Are you working on this? I'm currently on Debian and I can jump on it.
